controller/src/api/external/routes/tenant.rs

```diff
@@ -1,29 +1,32 @@
 use route_recognizer;
 use rusqlite::Connection;
-use std::io;
 use std::str::FromStr;
 use std::sync::mpsc::Sender;
 use tracing::{event, Level};
 
-use crate::api;
+use super::HttpResult;
 use crate::api::external::services::element::elements_set_right_name;
 use crate::api::types::element::OnlyId;
 use crate::api::types::tenant::Tenant;
-use crate::api::ApiChannel;
+use crate::api::{ApiChannel, RikError};
 use crate::database::RikRepository;
 
 pub fn get(
     _: &mut tiny_http::Request,
     _: &route_recognizer::Params,
     connection: &Connection,
     _: &Sender<ApiChannel>,
-) -> Result<tiny_http::Response<io::Cursor<Vec<u8>>>, api::RikError> {
+) -> HttpResult {
     if let Ok(mut tenants) = RikRepository::find_all(connection, "/tenant") {
         tenants = elements_set_right_name(tenants.clone());
-        let tenants_json = serde_json::to_string(&tenants).unwrap();
+        let tenants_json = serde_json::to_string(&tenants).map_err(RikError::ParsingError)?;
         event!(Level::INFO, "tenants.get, tenants found");
         Ok(tiny_http::Response::from_string(tenants_json)
-            .with_header(tiny_http::Header::from_str("Content-Type: application/json").unwrap())
+            .with_header(
+                tiny_http::Header::from_str("Content-Type: application/json").map_err(|_| {
+                    RikError::Error("tiny_http::Header failed to add header".to_string())
+                })?,
```

Comment:
IMHO, this kind of error handling is not needed, the static string will always be successful and won't fail. The code behind this method makes use assume that this method is infallible. In case we want to make thing homogeneous, make this header as a `const`, and replace all usage in the API.